cves/kernel/CVE-2021-3490.yml

```diff
@@ -131,10 +132,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: true
```

Comment:
This should be more of a written answer per the instructions rather than a boolean